opentelemetry-sdk/src/opentelemetry/sdk/trace/sampling.py

```diff
@@ -97,11 +117,14 @@ def should_sample(
         self,
         parent_context: Optional["SpanContext"],
         trace_id: int,
-        span_id: int,
         name: str,
-        attributes: Optional[Attributes] = None,
+        attributes: Attributes = None,
         links: Sequence["Link"] = (),
-    ) -> "Decision":
+    ) -> "SamplingResult":
+        pass
+
+    @abc.abstractmethod
+    def get_description(self) -> str:
         pass
 
```

Comment:
Might be nice to have for debugging?
```suggestion
    def __str__(self) -> str:
        return self.get_description()
```

Reply:
I don't really want to have any implementation in the abstract class.